tests/Common/BranchBucketsTest.php

```diff
@@ -18,34 +18,38 @@ public function setUp()
 
     public function testDropAllDevBucketsWhenDropBranch()
     {
+        $metadataKey = Metadata::BUCKET_METADATA_KEY_ID_BRANCH;
+        $metadataProvider = Metadata::PROVIDER_SYSTEM;
+
         $devBranchClient = new DevBranches($this->_client);
+        $metadata = new Metadata($this->_client);
+
+        $description = get_class($this) . '\\' . $this->getName();
 
         $branchName1 = $this->generateBranchNameForParallelTest();
+        $devBucketName1 = sprintf('Dev-Branch-Bucket-' . sha1($description));
+
         $branchName2 = $this->generateBranchNameForParallelTest('second');
+        $devBucketName2 = sprintf('Second-Dev-Branch-Bucket-' . sha1($description));
 
         // cleanup
         $this->deleteBranchesByPrefix($devBranchClient, $branchName1);
-        $this->deleteBranchesByPrefix($devBranchClient, $branchName2);
-
         $branch1 = $devBranchClient->createBranch($branchName1);
-        $branch2 = $devBranchClient->createBranch($branchName2);
+        $devBranchBucketId1 = $this->initEmptyBucket($devBucketName1, Client::STAGE_IN, $description);
 
-        $description = get_class($this) . '\\' . $this->getName();
-        $devBucketName1 = sprintf('Dev-Branch-Bucket-' . sha1($description));
-        $devBucketId1 = 'in.c-' . $devBucketName1;
-
-        $devBucketName2 = sprintf('Second-Dev-Branch-Bucket-' . sha1($description));
-        $devBucketId2 = 'in.c-' . $devBucketName2;
+        $branch1TestMetadata = [
+            [
+                'key' => Metadata::BUCKET_METADATA_KEY_ID_BRANCH,
+                'value' => $branch1['id'],
+            ]
+        ];
 
-        try {
-            foreach ([$devBucketId1, $devBucketId2] as $devBranchBucketId) {
-                if ($this->_client->getBucket($devBranchBucketId)) {
-                    $this->_client->dropBucket($devBranchBucketId);
-                }
-            }
-        } catch (ClientException $e) {
-        }
```

Comment:
Nahradil jsem `initEmptyBucket`. Tady se to dropovalo, o kus níže ten bucket zakládal. Takhle je to na jeden řádek.

Reply:
Jojo to je lepsie, neviem preco som si neuvedomil ze mame ten `bucketExist` :